assets/wizards/dashboard/index.js

```diff
@@ -106,30 +101,28 @@ const Dashboard = ( { items } ) => {
 						</div>
 					) }
 					{ displayAuth ? (
-						<Card className={ classnames( 'newspack-dashboard-card', 'google-oauth2' ) }>
+						<>
 							{ userBasicInfo ? (
-								<div className="newspack-dashboard-card__contents">
-									<Icon icon={ plugins } />
-									<div className="newspack-dashboard-card__header">
+								<Card className="newspack-dashboard-card">
```

Comment:
Why a regular `Card` here—is it just so we can use the `strong` in the message? Wonder if we could adapt the `ButtonCard` to render `children` if it exists instead of (or in addition to) the `desc` prop?

Reply:
It's not a link so it doesn't make sense to use a `ButtonCard`. It's also temporary and the Google oauth connection will be moved elsewhere.